helm/etcd-kubernetes-resources-count-exporter/values.yaml

```diff
@@ -6,9 +6,9 @@ image:
   tag: "[[ .Version ]]"
 pod:
   user:
-    id: 1000
+    id: 0
```

Comment:
When I checked this issue, I couldn't find any solution other than this. Therefore, it is OK.